### ISSUE TITLE
webqsp half

### ISSUE THREAD

**Issue by yaowang1112** [malllabiisc/EmbedKGQA#68]
Do we need relation matching for webqsp half ?

**Comment by apoorvumang** [comment]
No, it is not needed. Please refer Table 5 in the paper for the ablation study that shows that performance decreases when doing relation matching for half KG.

**Comment by lihuiliullh** [comment]
The accuracy I got on webqsp half was 0.478. May I know how you got 53.2% accuracy? @apoorvumang 

Validation accuracy decreases to 0.468394 from 0.478508, 0 more epoch to check

**Comment by apoorvumang** [comment]
@lihuiliullh did you re-train the KG embeddings or use the pretrained checkpoint?

**Comment by lihuiliullh** [comment]
I just downloaded the code and ran it following the instruction. 

![image](https://user-images.githubusercontent.com/53407011/144271571-bc4d175a-b3f3-4e19-bb2e-6984fc60781c.png)
